### PR TITLE
Added Login Modal and merged your auth-modal

### PR DESCRIPTION
**Pull request by Mihiirr**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -67,11 +67,13 @@
     "remix": "^1.2.3",
     "remix-auth": "^3.2.1",
     "remix-auth-google": "^1.1.0",
+    "sharp": "^0.30.2",
```
Comment (tirthGajjar) [inline package.json:70]: remove sharp and ts-node from `dependencies`

File: src/app/components/NavBar/ProfileDropdown.tsx [hunk 2]
```diff
@@ -18,14 +18,20 @@ const ProfileDropdown = ({ user, navigationMenu }: Props) => {
...
+              className="h-8 w-8 rounded-full"
+              src={user?.profilePicture}
+              alt=""
+              referrerPolicy="no-referrer"
+            />
+          ) : (
+            <div className="flex h-8 w-8 items-center justify-center rounded-3xl bg-blue-500 font-semibold text-white">
+              {user.name?.charAt(0)}
+            </div>
+          )}
```
Comment (tirthGajjar) [inline src/app/components/NavBar/ProfileDropdown.tsx:34]: create a reusable component

File: src/app/components/common/Modal.tsx [hunk 3]
```diff
@@ -56,68 +75,83 @@ export const Modal = ({ isOpen, setIsOpen }: ModalProps) => {
...
                   >
-                    Cancel
-                  </button>
+                    <div className="hidden">
+                      <Label htmlFor="redirect-to">redirect to</Label>
+                      <Input
+                        id="redirect-to"
+                        name="redirectTo"
+                        type="text"
+                        readOnly
```
Comment (tirthGajjar) [inline src/app/components/common/Modal.tsx:122]: the content inside the modal should have been a component

**Review by tirthGajjar — CHANGES_REQUESTED** [review]
Added a few comments

File: src/app/components/NavBar/ProfileDropdown.tsx (hunk 2 above)
Comment (Mihiirr) [reply src/app/components/NavBar/ProfileDropdown.tsx:34]: done

File: package.json (hunk 1 above)
Comment (Mihiirr) [reply package.json:70]: done

**Pull request closed without merging**